testdata/reconcile/test_currency/0/candidates.diff

```diff
@@ -0,0 +1,20 @@
+<journal-dir>/journal.beancount
```

Comment:
Upon closer inspection I realized that `mint_test.py` might be a more appropriate place to put the currency test. Could you move your test code into it, as well as the test artifacts under `testdata/source/mint/test_currency`.

Sorry I misguided you initially.

Reply:
moved tests to the appropriate folder